packages/atomic/src/locales.json

```diff
@@ -1266,8 +1266,8 @@
     "fr": "seulement {{count}} étoiles sur {{count}}"
   },
   "stars-range": {
-    "en": "{{value}} to {{count}} stars",
-    "fr": "{{value}} à {{count}} étoiles"
+    "en": "{{value}} out of {{count}} stars & up",
+    "fr": "{{value}} à {{count}} étoiles & plus"
```

Comment:
This is for the last 4 facet values displayed here:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/54454747/181094802-869cf628-da49-434a-a7cf-637368e491ea.png">

Last time, we decided not to fix this, but this success criterion is meant for users to be able to search aria labels based on what they see on the screen, and I think changing the labels to this doesn't look too bad.

More details here: https://www.w3.org/WAI/WCAG22/Techniques/failures/F96.html

Opinion?

@jpmarceau @y-lakhdar

Reply:
Looks good to me 🙂👌